### DISCUSSION TITLE
Adding a new component by drag and drop does not work.

### DISCUSSION THREAD

**Discussion by EricTheMagician** [mui/toolpad#3195] [Help and support]
Hi,

I'm trying out toolpad right now and it's not going very well.
I've create a new toolpad app.

I'm trying to add a new component to my canvas by drag and drop and it's not working. The "Drop component here" responds with a solid border when I bring in a component, but when I release the mouse click, nothing happens.

When I check the javascript console, there's nothing there.
It could be a bug, but I think there's a PEBKAC issue and I'm I would like to dive more in to getting logs and confirming that it's not a PEBKAC problem  before opening an issue.

I'm currently using chromium on linux with wayland. The distro is nixos.

**Comment by apedroferreira** [comment]
Hi Eric, thanks for trying out Toolpad and reporting this issue!
Does this happen on a newly created Toolpad application with a blank page, when placing your first component?
Or if it's something more specific, could you please provide a reproduction?

**Reply by EricTheMagician** [reply]
Hi @apedroferreira 
It does happen on a newly created toolpd application with a blank page when trying to place my first component.

I used yarn to create the project if that makes a difference.
